asserts/snap_asserts.go

```diff
@@ -462,7 +582,7 @@ func (snaprev *SnapRevision) checkConsistency(db RODatabase, acck *AccountKey) e
 	if err != nil {
 		return err
 	}
-	_, err = db.Find(SnapDeclarationType, map[string]string{
+	a, err := db.Find(SnapDeclarationType, map[string]string{
```

Comment:
I guess that this `a` is an assertion, but it would be nicer if it was written explicitly. If there are naming clashes that prevent it from being called `assertion`, I'd rather call it `foundAssertion` or `snapAssertion` than just `a`.

Reply:
we use "a" for assertion in many places here already

Reply:
OK